spec/filters/grok_spec.rb

```diff
@@ -648,8 +647,10 @@
     require 'tmpdir'
     require 'tempfile'
 
+    let(:pattern_dir) { File.join(LogStash::Environment::LOGSTASH_HOME, "patterns") }
+
     before do
-      pattern_dir = File.join(LogStash::Environment::LOGSTASH_HOME, "patterns")
+      FileUtils.mkdir(pattern_dir) unless File.exist?(pattern_dir)
```

Comment:
I think is basically doing the same, but will change ... semantically speaking makes sense.